scripts/golangci-lint.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
Removed the errcheck and gofmt check scripts as golangci lint actually can perform both.